playbooks/common/openshift-node/config.yml

```diff
@@ -109,6 +109,15 @@
     with_items: "{{ groups.oo_nodes_to_config | default([]) }}"
     when: hostvars[item].openshift.common.is_containerized | bool and (item in groups.oo_nodes_to_config and item in groups.oo_masters_to_config)
 
+- name: Configure node dnsmasq
+  hosts: oo_nodes_to_config
+  vars:
+    dns_domain: "{{ openshift.common.dns_domain }}"
+    kube_svc_ip: "{{ openshift.common.kube_svc_ip }}"
+  roles:
+    - role: openshift_node_dnsmasq
+      when: openshift_node_dnsmasq | default(True if openshift.common.version_gte_3_2_or_1_2 | bool else False) | bool
```

Comment:
Would it make more sense to include this as a dependency of openshfit_node through meta/main.yml instead? That would also eliminate the need to invoke openshift_facts from within the openshift_node_dnsmasq role, since it can be ordered after the openshift_common role has been run.